cli/cli_app.go

```diff
@@ -224,40 +224,47 @@ func runTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 	return runTerragruntWithConfig(terragruntOptions, terragruntConfig, false)
 }
 
-func processHooks(hooks []config.Hook, terragruntOptions *options.TerragruntOptions, previousExecError ...error) error {
+func processHooks(hooks []config.Hook, terragruntOptions *options.TerragruntOptions, previousExecErrors errors.MultiError) error {
+
 	if len(hooks) == 0 {
```

Comment:
Wouldn't it be a lot simpler to fix this by just checking if there are any non-`nil` errors?